src/main/java/org/prebid/server/bidder/telaria/TelariaBidder.java

```diff
@@ -64,14 +63,13 @@ public TelariaBidder(String endpointUrl, JacksonMapper mapper) {
         String seatCode = null;
         final BidRequest.BidRequestBuilder requestBuilder = bidRequest.toBuilder();
```

Comment:
I know this is not in scope of this pr, but please, move this field closer to its usage.